components/eventing-controller/pkg/handlers/beb.go

```diff
@@ -57,7 +52,13 @@ func getWebHookAuth(cfg env.Config) *types.WebhookAuth {
 }
 
 // SyncBebSubscription synchronize the EV2 subscription with the EMS subscription. It returns true, if the EV2 subscription status was changed
-func (b *Beb) SyncBebSubscription(subscription *eventingv1alpha1.Subscription, apiRule *apigatewayv1alpha1.APIRule, cleaner eventtype.Cleaner) (bool, error) {
+func (b *Beb) SyncSubscription(subscription *eventingv1alpha1.Subscription, cleaner eventtype.Cleaner, params ...interface{}) (bool, error) {
+	apiRule, ok := params[0].(*apigatewayv1alpha1.APIRule)
+	if !ok {
+		err := fmt.Errorf("failed to get apiRule from params[0]: %v", params[0])
+		b.Log.Error(err, "wrong parameter for subscription", "name:", subscription.Name)
```

Comment:
More general question: Is the colon behind `"name:"` our style of structured logging?

Reply:
currently there is no common way for structured logging. feel free to establish a structured logging standard in our components.